articles/openshift/howto-deploy-java-eap-app.md

```diff
@@ -277,31 +288,31 @@ Before deploying the application, let's creating the expected secret that will h
 
 2. Open the OpenShift console and navigate to the developer view (in the `</> Developer` perspective in the left hand menu)
 
     ![OpenShift console developer view](./media/howto-deploy-java-eap-app/console_developer_view.png)
 
-3. Once you are in the `</> Developer` perspective, ensure you have selected the `eap-demo` project at the projects combo box.
+3. Once you are in the `</> Developer` perspective, ensure you have selected the `eap-demo` project at the `Project` combo box.
 
     ![OpenShift console project combo box](./media/howto-deploy-java-eap-app/console_project_combo_box.png)
 
 4. Click on `+Add`, then Helm Chart. You will arrive at the Helm Chart catalog available on your ARO cluster. Write `eap` on the filter input box to filter all the Helm Charts and get the EAP ones. At this stage, you should see two options:
 
     ![OpenShift console EAP Helm Charts](./media/howto-deploy-java-eap-app/console_eap_helm_charts.png)
 
-5. Since our application uses Microprofile capabilities, you have to select for this demo the one named EAP Xp (at the time of this writing, the exact version is EAP Xp 1.0.0). The `Xp` stands for Expansion Pack. With the JBoss Enterprise Application Platform expansion pack, developers can use Eclipse MicroProfile application programming interfaces (APIs) to build and deploy microservices-based applications.
+5. Since our application uses MicroProfile capabilities, you have to select for this demo the Helm Chart for EAP XP (at the time of this writing, the exact version of the Helm Chart is EAP Xp3 v1.0.0). The `Xp3` stands for Expansion Pack version 3.0.0. With the JBoss Enterprise Application Platform expansion pack, developers can use Eclipse MicroProfile application programming interfaces (APIs) to build and deploy microservices-based applications.
 
-6. Click on the `EAP Xp` Helm Chart and then `Install Helm Chart` button.
+6. Click on the `EAP Xp` Helm Chart, and then on `Install Helm Chart` button.
 
 At this point, we need to configure the chart to be able to build and deploy the application:
 
 1. Change the name of the release to `eap-todo-list-demo`. 
-1. We can configure the Helm Chart either using a `Form View` or a `YAML View`. Check `YAML View` radio button in the `Configure via` box.
+1. We can configure the Helm Chart either using a `Form View` or a `YAML View`. Check the `YAML View` radio button in the `Configure via` box.
 1. Then, change the YAML content to configure the Helm Chart by copying the content of the Helm Chart file available at `todo-list/deployment/application/todo-list-helm-chart.yaml` instead of the existing content:
 
    ![OpenShift console EAP Helm Chart YAML content](./media/howto-deploy-java-eap-app/console_eap_helm_charts_yalm_content.png)
```

Comment:
I can see here "console_eap_helm_charts_yalm_content" - I think this should have yaml instead of yalm and I think https://github.com/yersan/azure-docs/blob/yborgess-aro-eap-howto/articles/openshift/media/howto-deploy-java-eap-app/console_eap_helm_charts_yalm_content.png needs to be renamed accordingly?